crmsh/utils.py

```diff
@@ -3107,4 +3109,39 @@ def time_value_with_unit(time_value):
     Check if the time value contains unit
     """
     return re.search(r'^\d+[a-z]+$', time_value) is not None
+
+
+
+def ansible_installed():
+    return shutil.which('ansible')
+
+
+def ansible_facts(module_name) -> dict:
+    proc = subprocess.Popen(['ansible', '-m', module_name, 'localhost'],
+                            shell=False,
+                            stdin=None,
+                            stdout=subprocess.PIPE,
+                            stderr=subprocess.PIPE)
+    out, err = proc.communicate(None)
+    proc.wait()
+    rc = proc.returncode
+    out = out.decode('utf-8')
+    err = err.decode('utf-8')
+
+    if rc == -1:
+        logger.error(f'Calling ansible: {err}')
```

Comment:
This will break if there is a `%` in `str(err)`.

```suggestion
        logger.error('Calling ansible: %s', err)
```